src/cc/frontends/clang/kbuild_helper.h

```diff
@@ -21,6 +21,8 @@
 #include <errno.h>
 #include <ftw.h>
 
+#define PROC_KHEADERS_PATH "/proc/kheaders.tar.xz"
```

Comment:
I thought lkml requested /sys/kernel/tracing/ instead?
The rest looks good.
let's land it when that bit is certain.

Reply:
I think there was no agreement on that and any location seemed ok. How do we resolve this decision?

Reply:
From what I read, to relieve Olof's concerns, /sys/ directory (not sure about /sys/kernel/tracing) is preferred? It looks a a few other folks like Steve is also on board on this.

Reply:
Ok. It's a confusing discussion. I guess will target sys/kernel/ and resend then.. wdyt? Greg kh was on board with that. thanks.

Reply:
Since Greg gave a reviewed by tag, I think he's onboard with proc as well.  What do you guys prefer? If proc, is ok please give Ack or Reviewed-by tag on lkml posting. Thanks!